### PR TITLE
Feature fix usart

### PR DESCRIPTION
**Pull request by JasperOosterbroek**
Fix release with new usart library

### CODE REVIEW

File: code/headers/uarm_swift_pro.hpp [hunk 1]
```diff
@@ -16,19 +16,20 @@ namespace r2d2::robot_arm {
...
          */
-        uarm_swift_pro_c(const r2d2::hardware_usart_c &usart_bus);
+        uarm_swift_pro_c(usart_c &usart_bus);
         /**
          * robot arm constructor, creates a usart bus from the bautrate and port
          **/
-        uarm_swift_pro_c(unsigned int &bautrate,
-                         r2d2::uart_ports_c &usart_port);
+        //  uarm_swift_pro_c(unsigned int &bautrate,
+        //                 r2d2::usart::uart_ports_c &usart_port);
```
Comment (SjoerddzHu) [inline code/headers/uarm_swift_pro.hpp]: commented uit code will not be approved when merging with the master.

File: code/src/uarm_swift_pro.cpp [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+    uarm_swift_pro_c::uarm_swift_pro_c(usart_c &usart_bus)
         : usart_bus(usart_bus) {
     }
-    uarm_swift_pro_c::uarm_swift_pro_c(unsigned int &bautrate,
-                                       r2d2::uart_ports_c &usart_port)
-        : usart_bus(bautrate, usart_port) {
-    }
+
+    // uarm_swift_pro_c::uarm_swift_pro_c(unsigned int &bautrate,
+    //                                 r2d2::usart::uart_ports_c &usart_port)
```
Comment (SjoerddzHu) [inline code/src/uarm_swift_pro.cpp]: commented out code will not be approved when merging with the master branche.

**Review by SjoerddzHu — CHANGES_REQUESTED** [review]
see the comments within the submits.

File: code/src/uarm_swift_pro.cpp (hunk 2 above)
Comment (JasperOosterbroek) [reply code/src/uarm_swift_pro.cpp]: fixed in https://github.com/R2D2-2019/robot_arm/commit/654c3de55e13a57ff32a750c14c4098c7a6bed7b

File: code/headers/uarm_swift_pro.hpp (hunk 1 above)
Comment (JasperOosterbroek) [reply code/headers/uarm_swift_pro.hpp]: fixed in https://github.com/R2D2-2019/robot_arm/commit/654c3de55e13a57ff32a750c14c4098c7a6bed7b

**Review by SjoerddzHu — APPROVED** [review]

**Pull request merged**